src/lib.rs

```diff
@@ -8,6 +8,7 @@
 // except according to those terms.
 
 #![deny(missing_docs)]
+#![allow(unknown_lints)]
```

Comment:
What does this do?

Reply:
The Rust compiler doesn't know clippy's lints and otherwise shows a warning for the `allows` in other places.